### PR TITLE
 Generate manual pages

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
### Genarate manual pages:
`go run doc/main.go --man-page`
will generate exo doc manual page in `manpage` folder

### Goreleaser
packaged in goreleaser

### CODE REVIEW

**Comment by pierre-emmanuelJ** [comment]
For you @vincentbernat 😸

File: .goreleaser.yml [hunk 1]
```diff
@@ -44,6 +44,14 @@ nfpm:
     - deb
     - rpm
 
+  overrides:
+    deb:
+      files:
+        "./manpage/exo*.gz": "/usr/share/man/man1"
+    rpm:
+      files:
+        "./manpage/exo*.gz": "/usr/share/man/man1"
```
Comment (greut) [inline .goreleaser.yml]: <3 !

**Comment by greut** [comment]
Is there a way to squeeze the auto-completion back in there too?

**Comment by pierre-emmanuelJ** [comment]
yes it's possible 👍

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@ Dockerfile.goreleaser
 go.sum
 ops.asc
 exo
+.DS_Store
```
Comment (greut) [inline .gitignore]: :green_apple:

File: .travis.yml [hunk 3]
```diff
@@ -33,6 +33,7 @@ jobs:
       script:
         - go run -mod vendor doc/main.go --man-page
         - cd manpage && gzip * && cd ..
+        - go run -mod vendor completion/main.go
```
Comment (greut) [inline .travis.yml]: let's put this into the repository

Comment (greut) [reply .travis.yml]: checking in travis that it doesn't change is good though

File: .goreleaser.yml [hunk 4]
```diff
@@ -27,6 +27,7 @@ archive:
       format: zip
   files:
     - LICENSE
+    - bash_completion
```
Comment (greut) [inline .goreleaser.yml]: and the manpages ?

Comment (greut) [reply .goreleaser.yml]: asking for a friend.

Comment (pierre-emmanuelJ) [reply .goreleaser.yml]: 😆

File: .gitignore (hunk 2 above)
Comment (brutasse) [reply .gitignore]: It's also something that can go in global `.gitignore` :)

**Comment by greut** [comment]
```console
% sudo dpkg -i cli_*.deb
```

screenshot from 2018-11-13 10-59-00

screenshot from 2018-11-13 11-00-29

**Comment by greut** [comment]
try it!

- https://sos-ch-dk-2.exo.io/yoanyoan/cli_SNAPSHOT-662c16146f0a58a095296a1d5e702262ad894cf3_linux_amd64.deb
  3b189db87f0b657b94d5d45ccfea88c5bc30a68b54cbf352bea1121a47c627a5  cli_SNAPSHOT-662c16146f0a58a095296a1d5e702262ad894cf3_linux_amd64.deb

- https://sos-ch-dk-2.exo.io/yoanyoan/cli_SNAPSHOT-662c16146f0a58a095296a1d5e702262ad894cf3_linux_amd64.rpm
  5e106f2805241251610933a647297a6d72df0fa522d26a481f6e7da3d0c0f7fa  cli_SNAPSHOT-662c16146f0a58a095296a1d5e702262ad894cf3_linux_amd64.rpm

**Pull request merged**